AppDB/appscale/datastore/fdb/__init__.py

```diff
@@ -0,0 +1 @@
+from appscale.datastore.fdb.fdb_datastore import FDBDatastore
```

Comment:
It was just to make the import prettier (`from .fdb import FDBDatastore` rather than `from .fdb.fdb_datastore import FDBDatastore`). You're right, though, it's not necessary.